environment.rb

```diff
@@ -0,0 +1,140 @@
+module Environment
+    class Depot
```

Comment:
Indexed tabs in Ruby are usually two spaces, if you configure your IDE to set the tab key like this sometimes Github just take it as standard tab; I use raw two spaces instead of the tab key, that may help to get the right tabbing here.